src/Fabulous.XamarinForms/Views/Controls/Button.fs

```diff
@@ -47,9 +104,120 @@ type ButtonModifiers =
         )
 
     [<Extension>]
-    static member inline font(this: WidgetBuilder<'msg, #IButton>, value: double) =
+    static member inline textTransform(this: WidgetBuilder<'msg, #IButton>, value: TextTransform) =
+        this.AddScalar(Button.TextTransform.WithValue(value))
+
+    [<Extension>]
+    static member inline cornerRadius(this: WidgetBuilder<'msg, #IButton>, value: int) =
+        this.AddScalar(Button.CornerRadius.WithValue(value))
+
+    [<Extension>]
+    static member inline borderColor(this: WidgetBuilder<'msg, #IButton>, light: Color, ?dark: Color) =
+        this.AddScalar(
+            Button.BorderColor.WithValue(
+                { Light = light
+                  Dark =
+                      match dark with
+                      | None -> ValueNone
+                      | Some v -> ValueSome v }
+            )
+        )
+
+    [<Extension>]
+    static member inline borderWidth(this: WidgetBuilder<'msg, #IButton>, value: float) =
+        this.AddScalar(Button.BorderWidth.WithValue(value))
+
+    [<Extension>]
+    static member inline padding(this: WidgetBuilder<'msg, #IButton>, value: Thickness) =
+        this.AddScalar(Button.Padding.WithValue(value))
+
+    [<Extension>]
+    static member inline padding(this: WidgetBuilder<'msg, #IButton>, value: float) =
+        ButtonModifiers.padding (this, Thickness(value))
+
+    [<Extension>]
+    static member inline padding
+        (
+            this: WidgetBuilder<'msg, #IButton>,
+            left: float,
+            top: float,
+            right: float,
+            bottom: float
+        ) =
+        ButtonModifiers.padding (this, Thickness(left, top, right, bottom))
+
+    [<Extension>]
+    static member inline characterSpacing(this: WidgetBuilder<'msg, #IButton>, value: float) =
+        this.AddScalar(Button.CharacterSpacing.WithValue(value))
+
+    [<Extension>]
+    static member inline contentLayout(this: WidgetBuilder<'msg, #IButton>, value: ContentLayout) =
```

Comment:
For this one, I would simply do
```suggestion
    static member inline contentLayout(this: WidgetBuilder<'msg, #IButton>, position: Xamarin.Forms.Button.ButtonContentLayout.ImagePosition, spacing: float) =
```

That way we don't need additional structs and constructing/deconstructing tuples.
```fs
Button("...", Clicked)
    .contentLayout(ImagePosition.Top, 10.f)
```

Reply:
Yeah Agreed.